common/types/ballot.go

```diff
@@ -71,6 +71,8 @@ type Ballot struct {
 	EligibilityProofs []VotingEligibility `scale:"max=25000"`
 	// from the smesher's view, the set of ATXs eligible to vote and propose block content in this epoch
 	// only present in smesher's first ballot of the epoch
+	// this field isn't actually used any more (replaced by InnerBallot.EpochData)
+	// TODO (mafa): remove this field in Ballot v2
```

Comment:
Then perhaps we should set the scale max to 0? AFAIR, the database was pruned (including blobs) so it should be safe.

Reply:
We can consider this for `v1.6.0`. I would like to test if a full sync with this set to 0 breaks anything before merging it.